### PR TITLE
Added a draft of InsertColumns()

### PR DESCRIPTION
**Pull request by xaionaro**
Added a draft of InsertColumns()

Relates: https://github.com/go-reform/reform/issues/30

### CODE REVIEW

**Comment by xaionaro** [comment]
Oops. Sorry. I was going to create a PR to my own repository to make some test before making this PR.

File: querier_commands.go [hunk 1]
```diff
@@ -74,6 +86,66 @@ func (q *Querier) Insert(str Struct) error {
...
+	}
+
+	return q.insert(str, columns, values)
+}
+
+// InsertColumns inserts a struct into SQL database table with specified columns list to be explicitly set
+// If str implements BeforeInserter, it calls BeforeInsert() before doing so.
+//
+// It fills record's primary key field.
+func (q *Querier) InsertColumns(str Struct, columns ...string) error {
```
Comment (AlekSi) [inline querier_commands.go]: LGTM.

**Comment by AlekSi** [comment]
If you can, please do not do refactoring and implement new feature in the same commit.

**Comment by xaionaro** [comment]
@AlekSi, well, I can recreate all commits (it's not lazy or hard to me). Am I understand right, that I need to make 4 commits in this case?
* refactoring of old code
* adding new feature
* adding "GroupID" to model (for DEFAULT-value related checks) and `reform` it
* adding a test for InsertColumns()

**Comment by xaionaro** [comment]
@AlekSi, what should I fix to pass appveyor test?

File: internal/test/sql/mysql_init.sql [hunk 2]
```diff
@@ -4,6 +4,7 @@ CREATE TABLE people (
   id int NOT NULL AUTO_INCREMENT,
   name varchar(255) NOT NULL,
   email varchar(255),
+  group_id int(1) DEFAULT 65534,
```
Comment (AlekSi) [inline internal/test/sql/mysql_init.sql]: 1?

**Comment by AlekSi** [comment]
> * refactoring of the old code
> * adding the new feature
> * adding field "GroupID" to the model (for DEFAULT-value related checks) and reform it
> * adding a test for InsertColumns()

That's a very good order! Ideal would be:
* refactoring of the old code
* adding field "GroupID" to the model (for DEFAULT-value related checks) and reform it
* InsertColumns() and test

Can you also do a rebase?
AppVeyor is not yet configured, should be ignored for now.

File: internal/test/models/good.go [hunk 3]
```diff
@@ -14,6 +14,7 @@ type (
 	//reform:people
 	Person struct {
 		ID        int32      `reform:"id,pk"`
+		GroupID   int32      `reform:"group_id"`
```
Comment (AlekSi) [inline internal/test/models/good.go]: Column `group_id` NULLable in the database, this field should be a pointer.

**Comment by xaionaro** [comment]
> That's a very good order! Ideal would be:

Ok. :)

> Can you also do a rebase?

I was going to just use a force push into my repository. But I can try to polish the commit history using rebase (instead of force push) if required.

**Comment by AlekSi** [comment]
I mean I just pushed a small change to `v1-stable`, it can affect your work.

**Comment by xaionaro** [comment]
@AlekSi, I force pushed a new commit history. Please check :)

**Comment by AlekSi** [comment]
Moved to #39.

**Pull request closed without merging**